generator/internal/scaffold_generator.cc

```diff
@@ -60,6 +60,15 @@ ProductPath ParseProductPath(std::string const& product_path) {
   return make_result(std::prev(v.end()));
 }
 
+std::string FormatCloudServiceDocsLink(
+    std::map<std::string, std::string> const& vars) {
+  return absl::StrCat("[cloud-service-docs]: ",
+                      vars.find("documentation_uri") == vars.end()
+                          ? "https://cloud.google.com/$site_root$ [VERIFY HERE]"
```

Comment:
Elsewhere we use `[EDIT HERE]`. It seems sad that we have to search for two different strings.